Documentation/concepts/observability/hubble-configuration.rst

```diff
@@ -100,21 +103,28 @@ is used to schedule for certificates regeneration (regardless of their expiratio
 Auto generated certificates via cert-manager
 --------------------------------------------
 
-This method rely on `cert-manager <https://cert-manager.io/>`__ to generate certificate.
-``cert-manager`` now becomes de-facto way to manage TLS on k8s, and compared to above methods,
-it has the following advantages:
-
-* No need for extra cronJob
-* Support multiple issuers: CA, Vault, Let's Encrypt, Google CAS,...
-  You can choose the issuer that fits your organization's requirements.
-* Manage certs via a CRD, which is more straightforward than inspecting the PEM file.
-* Auto-renew certificates
+This method rely on `cert-manager <https://cert-manager.io/>`__ to generate the
+TLS certificates. cert-manager has becomes the de-facto way to manage TLS on
+Kubernetes, and it has the following advantages compared to the previously
+documented methods:
+
+* Support multiple issuers (e.g. a custom CA,
+  `Vault <https://www.vaultproject.io/>`__,
+  `Let's Encrypt <https://letsencrypt.org/>`__,
+  `Google's Certificate Authority Service <https://cloud.google.com/certificate-authority-service>`__,
+  and more) allowing to choose the issuer fitting your organization's
+  requirements.
+* Manages certificates via a
+  `CRD <https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/>`__
+  which is easier to inspect with Kubernetes tools than PEM file.
 
 **Installation steps**:
 
-1. First install `cert-manager <https://cert-manager.io/docs/installation/>`__ and setup `issuer <https://cert-manager.io/docs/configuration/>`_.
-   Please make sure that your issuer is be able to create certificates under the ``cilium.io`` domain name.
-2. Install/upgrade cilium with bellow configs:
+1. First, install `cert-manager <https://cert-manager.io/docs/installation/>`__
+   and setup an `issuer <https://cert-manager.io/docs/configuration/>`_.
+   Please make sure that your issuer is be able to create certificates under the
+   ``cilium.io`` domain name.
+2. Install/upgrade Cilium including the following Helm flags:
```

Comment:
nit: I'm not so worried about this right here since I don't expect these instructions to change much, but you can use `#.` at the start of the lines to enumerate the numbers for you, and then as long as you keep the indentation correct, sphinx will automatically figure out the correct next `#.` number to place when rendering the pages. Can save on reworking the numbers in numbered lists that we expect to change over time.

Reply:
Marking as resolved as we don't enumerate the options anymore with the tabbed section.

Reply:
:thinking: The numbered lists appear to still be here, the start of the line that this comment is based on has `2.`.